scenario_player/setup/nodes/config.py

```diff
@@ -0,0 +1,139 @@
+"""Helper module for configuring, starting and stopping Raiden Clients."""
+import pathlib
+import socket
+from typing import TYPE_CHECKING, Optional
+
+import structlog
+from eth_utils.address import to_checksum_address
+
+from scenario_player.setup.nodes.utils import create_keystore
+
+if TYPE_CHECKING:
+    from eth_utils.typing import ChecksumAddress
+    from scenario_player.setup.nodes.utils import RaidenBinary
+    from scenario_player.scenario import ScenarioYAML
+
+log = structlog.getLogger(__name__)
+
+
+class RaidenConfig:
+    """Raiden Client flag options.
+
+    Exposes common attributes of the client, and offers simplified
+    configuration options.
+
+    Exposes a :meth:`.as_command` method, which converts its attributes into
+    a :class:`scenario_player.setup.nodes.executor.RaidenClientExecutor` compatible
+    command list.
+    """
+
+    def __init__(self, loaded_yaml: ScenarioYAML, index: int, chain: str, client_addr: str, run_number: int):
+        self._yaml = loaded_yaml
+        self._chain = chain
+        self.index = index
+        self.run_number = run_number
+        self._eth_address = client_addr
+        self._netloc = None
+
+    @property
+    def eth_address(self) -> ChecksumAddress:
+        """The ethereum address of the account being managed by this instance.
+
+        TODO: Make this available to be settable in and gettable from the scenario yaml.
```

Comment:
That's not possible, the address is determined by the private key in the keystore.

Reply:
Currently, yes - I figured it might be useful when we revert the "always fresh accounts"  patch.

Reply:
But even then the address is an inherent property resulting from the private key and can't be set independent of that.